### ISSUE TITLE
support for add_dic

### ISSUE THREAD

**Issue by tmccombs** [thomas-joiner/HunspellBridJ#4]
Add support for the add_dic method.

**Comment by thomas-joiner** [comment]
Unfortunately, the `add_dic` method is only supported through the C++ interface.

This library uses the C interface exposed by the library.

I would not be averse to converting to use the C++ interface, however, I can not find suitable documentation on how to create one, and my attempts to use JNAerator to generate a C++ interface failed.

If you could find sufficient documentation or supply code yourself, I would be happy to include it.

**Comment by tmccombs** [comment]
From line 63 of [hunspell.h](https://github.com/hunspell/hunspell/blob/master/src/hunspell/hunspell.h):

```java
LIBHUNSPELL_DLL_EXPORTED int Hunspell_add_dic(Hunhandle* pHunspell,
                                              const char* dpath);
```

I'm working on a pull request, I'll send it to you when I'm done.

**Comment by thomas-joiner** [comment]
Okay, I'll be happy to accept your pull request.

I figured out what was confusing me. For some reason the versions that I built and added (for the Windows versions) ended up being 1.3.3. (Probably due to the fact that I was having issues compiling 1.3.4 at first, and ended up going back a version to see if that was the issue.) Since I was looking at the exported functions of the DLL, the function was not there yet.

I have updated the versions to actually be 1.3.4 and pushed to GitHub. If you are testing on Windows, you will want to pull first, otherwise it will be unable to bind to the function as it will not exist.
